packages/styling/src/styles/getFocusStyle.ts

```diff
@@ -21,6 +21,7 @@ export function getFocusStyle(
   return mergeStyles(
     {
       outline: 'transparent',
+      '::-moz-focus-inner': { border: 0 },
```

Comment:
why is this needed? What behavior does it change?

Also, should autoprefixing be adding this rule if outline transparent is present?